src/index.html

```diff
@@ -5,10 +5,28 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700&family=Montserrat:wght@400;600&family=Roboto:wght@400;700&display=swap" rel="stylesheet">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <!-- <h1>Moyo header</h1> -->
+    <header class="header">
+      <a class="logo" href="#"><img src="images/logo.png" alt="logo"></a>
```

Comment:
```suggestion
      <a class="logo" href="#">
        <img src="images/logo.png" alt="logo">
      </a>
```
Consider fixing indents according to the code style guide

Reply:
Still not fixed from the previous review